### ISSUE TITLE
A stored injection vulnerability

### ISSUE THREAD

**Issue by jgj212** [WebsiteBaker/WebsiteBaker#2]
Hello:
   My team found a stored injection vulnerability in "/account/details.php"
   ![1](https://cloud.githubusercontent.com/assets/1484292/25955693/835d4406-369c-11e7-95a8-12f45bf5d011.png)

   Line 22 has a var named $display_name which can be controlled by browser side.
   Line 47 wirte $display_name to database with escapeString.
   
   After checking the html, we found that $display_name can bypass escapeString.
   ![2](https://cloud.githubusercontent.com/assets/1484292/25955812/c7ad3346-369c-11e7-8704-227e6d314420.png)

   Testing as follow,
   URL: http://localhost/websitebaker/account/preferences.php
   POST：action=details&display_name=233" onfocus="alert(/xss/)" autofocus="123

   When admintrator view the user info, xss will occur
   ![3](https://cloud.githubusercontent.com/assets/1484292/25956209/af76a022-369d-11e7-90d7-8f84b4036049.png)

Credit: ADLab of VenusTech

**Comment by DarkViper** [comment]
Sorry, but this repository is from the Stone Age. More then 4 years old, long time outdated and never supported by us..
I think it is more useful to visit our original repository of WebsiteBaker.
http://project.websitebaker.org/
http://forum.websitebaker.org/
Manuela

**Comment by jgj212** [comment]
@DarkViper  Yes,  i  downloaded WebsiteBaker 2.10.0 from http://wiki.websitebaker.org/doku.php/en/downloads. I don't know where to submit issue, so I just submit here.
